pkg/apis/config/testdata/feature-flags-bundles-and-custom-tasks.yaml

```diff
@@ -7,3 +7,4 @@ data:
   enable-tekton-oci-bundles: "true"
   enable-custom-tasks: "true"
   enable-api-fields: "stable"
+  enable-step-actions: "true"
```

Comment:
Is this really needed here?

Reply:
shall remove it considering that api fields are anyways stable